demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -99,7 +101,34 @@ export function ProductsGrid({ filter, toolbarAction, rowAction }: Props): React
     const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("ProductsGrid") };
 
     const columns: GridColDef<GQLProductsGridFutureFragment>[] = [
-        { field: "inStock", headerName: intl.formatMessage({ id: "product.inStock", defaultMessage: "In stock" }), type: "boolean", width: 90 },
+        {
+            field: "inStock",
+            headerName: intl.formatMessage({ id: "product.inStock", defaultMessage: "In Stock" }),
+            type: "singleSelect",
+            valueOptions: [
+                { value: "true", label: intl.formatMessage({ id: "product.inStock.true.primary", defaultMessage: "In stock" }) },
+                { value: "false", label: intl.formatMessage({ id: "product.inStock.false.primary", defaultMessage: "Out of stock" }) },
+            ],
+            renderCell: ({ row }) => {
+                const valueLabels: Record<string, React.ReactNode> = {
+                    true: (
+                        <GridCellContent
+                            primaryText={intl.formatMessage({ id: "product.inStock.true.primary", defaultMessage: "In stock" })}
```

Comment:
You could use `<FormattedMessage>` here

Reply:
Done in [52ca9e5](https://github.com/vivid-planet/comet/pull/2422/commits/52ca9e50b07b11543fed87c42f4a2fe5812499b8)